bindings/wasm/examples/src/create_vp.js

```diff
@@ -46,7 +39,7 @@ async function createVP(clientConfig) {
 
     // In order to validate presentations and credentials one needs to resolve the DID Documents of 
     // the presentation holder and of credential issuers. This is something the `Resolver` can help with.  
-
+    
     const resolver = await new Resolver();
```

Comment:
Oh, `new Resolver()` defaults to the `Mainnet`. We should probably use `ResolverBuilder` everywhere so it uses the `clientConfig` from the examples.

E.g.
```suggestion
    const resolver = await new ResolverBuilder()
        .clientConfig(Config.fromNetwork(clientConfig.network))
        .build();
```